docs/13_SKILLS.md

```
@@ -107,8 +107,46 @@ end
+
+# User typed: /code-review focus on security
+if skills.activated?("code-review")
+  agent.generate("The code-review skill was invoked again — its instructions are above. focus on security")
+else
+  agent.generate(skills.activation_prompt("code-review", args: "focus on security"))
+end
+```
+
+`activation_prompt("code-review", args: "focus on security")` returns:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)